resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -0,0 +1,1537 @@
+- slug: general
+  name: General descriptor schema
+  description: General descriptor schema.
+  version: 1.0.0
+  schema:
+    - name: general
+      label: General
+      group:
+        - name: name
+          label: Sample name
+          type: basic:string
+          required: false
+        - name: slug
+          label: Sample slug
+          type: basic:string
+          required: false
+        - name: id
+          label: Sample id
+          type: basic:integer
+          required: false
+        - name: collection
+          label: Collection
+          type: basic:string
+          required: false
+        - name: species
+          label: Species
+          type: basic:string
+          default: Homo sapiens
+          required: false
+          choices:
+            - label: Caenorhabditis elegans
+              value: Caenorhabditis elegans
+            - label: Cricetulus griseus
+              value: Cricetulus griseus
+            - label: Dictyostelium discoideum
+              value: Dictyostelium discoideum
+            - label: Dictyostelium purpureum
+              value: Dictyostelium purpureum
+            - label: Drosophila melanogaster
+              value: Drosophila melanogaster
+            - label: Homo sapiens
+              value: Homo sapiens
+            - label: Macaca mulatta
+              value: Macaca mulatta
+            - label: Mus musculus
+              value: Mus musculus
+            - label: Odocoileus virginianus texanus
+              value: Odocoileus virginianus texanus
+            - label: Rattus norvegicus
+              value: Rattus norvegicus
+            - label: Solanum tuberosum
+              value: Solanum tuberosum
+            - label: Other
+              value: other
+        - name: annotator
+          label: Annotator
+          type: basic:string
+          required: false
+        - name: description
+          label: Description
+          type: basic:string
+          required: false
+    - name: non_clinical
+      label: Non-clinical information
+      group:
+        # - name: quality_status
+        #   label: Quality status
+        #   type: basic:string
+        #   required: false
+        #   choices:
+        #     - label: Pass
+        #       value: PASS
+        #     - label: Warning
+        #       value: WARNING
+        #     - label: Fail
+        #       value: FAIL
+        # - name: quality_message
+        #   label: Quality message
+        #   type: basic:string
+        #   required: false
+        - name: cell_line
+          label: Cell line information
+          group:
+            # - name: primary_cell_line
```

Comment:
we can enable primary_cell_line again if required, for now we commented it out. Rest of the fields should be sufficient to carry most of the cell line information 
cc @amiamrita

Reply:
Field `primary_cell_line` was removed and can be added in the future if needed. Changelog was updated accordingly.